crates/router/src/core/errors/user.rs

```diff
@@ -90,6 +90,10 @@ pub enum UserErrors {
     SSOFailed,
     #[error("profile_id missing in JWT")]
     JwtProfileIdMissing,
+    #[error("MaxTotpAttemptsReached")]
```

Comment:
Please match the string here with the message in `get_error_message` function, like we are doing in some of the above errors.